app/controllers/FaciaContentApiProxy.scala

```diff
@@ -3,27 +3,23 @@ package controllers
 import java.net.{URI, URLEncoder}
 
 import akka.actor.ActorSystem
-import auth.PanDomainAuthActions
 import com.amazonaws.auth.profile.ProfileCredentialsProvider
 import com.amazonaws.auth.{AWSCredentialsProviderChain, STSAssumeRoleSessionCredentialsProvider}
 import com.gu.contentapi.client.{IAMEncoder, IAMSigner}
-import conf.ApplicationConfiguration
 import metrics.FaciaToolMetrics
 import model.Cached
 import play.api.Logger
-import play.api.libs.ws.WSAPI
-import play.api.mvc._
 import switchboard.SwitchManager
 import util.ContentUpgrade.rewriteBody
 
 import scala.concurrent.ExecutionContext.Implicits.global
 
-class FaciaContentApiProxy(ws: WSAPI, val config: ApplicationConfiguration) extends Controller with PanDomainAuthActions {
+class FaciaContentApiProxy(val deps: BaseFaciaControllerComponents) extends BaseFaciaController(deps) {
   implicit class string2encodings(s: String) {
     lazy val urlEncoded = URLEncoder.encode(s, "utf-8")
   }
 
-  override lazy val actorSystem = ActorSystem()
+  lazy val actorSystem = ActorSystem()
```

Comment:
I don't think this is used? (at least according to IntelliJ)

Reply:
I don't think so either.